pkgs/by-name/de/debos/package.nix

```
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  dpkg,
+  pkg-config,
+  glib,
+  ostree,
+  qemu,
+  unzip,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "debos";
+  version = "1.1.5";
+
+  src = fetchFromGitHub {
+    owner = "go-debos";
+    repo = "debos";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-1Xv7r1uyISp1whALtFZWNQgTMH0swd/0+QdRCmXM4DQ=";
+  };
+
+  vendorHash = "sha256-FdcmitxGKzkHqFjllDGW24Lr8yrsYDBzuEDRco9CW14=";
+
+  nativeBuildInputs = [
+    dpkg
+    pkg-config
+    unzip
+  ];
+
+  buildInputs = [
+    glib
+    ostree
+    qemu
+  ];
+
+  doCheck = true;
+
```

```suggestion
  doCheck = true;

```

I think that tests are run by default.